ios/FluentUI/HUD/HUDView.swift

```diff
@@ -172,44 +188,13 @@ private class HUDIndicatorView: UIView {
             return activityIndicatorView
         case .success:
             let imageView = UIImageView(image: .staticImageNamed("checkmark-white-40x40"))
-            imageView.accessibilityLabel = "Accessibility.HUD.Done".localized
+            // unfortuantely, this resource image isn't the right size
```

Comment:
isn't *currently* the right size :). Can we indicate in the comment whether the fix is new artwork or design changes to the size?

Reply:
yes. definitely started a separate thread with design team.